test/unit/host_test.rb

```diff
@@ -1714,6 +1708,134 @@ def teardown
     assert_equal "fqdn=#{host.shortname}.yourdomain.net", LookupValue.find(lookup_value.id).match
   end
 
+  test '#setup_clone skips new records' do
+    assert_nil FactoryGirl.build(:host, :managed).send(:setup_clone)
+  end
+
+  test '#setup_clone is public and clones interfaces so delegated attributes are applied to cloned interfaces' do
+    host = FactoryGirl.create(:host, :managed)
```

Comment:
It's only two tests, one of which specifically tests *new* records and thus uses `.build` while the other uses `.create`. I don't think a context block adds any value here?